src/App.jsx

```diff
@@ -1,28 +1,57 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { todosPromis } from './api/todos';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
+    selectedUser: {},
+  };
+
+  componentDidMount() {
+    todosPromis()
+      .then(todos => (
+        this.setState({
+          todos: todos.filter(item => (item.title && item.userId)),
+        })
+      ));
+  }
+
+  currentUserId = (userId) => {
+    this.setState({ selectedUserId: userId });
+  };
+
+  clearUserinfo = () => {
+    this.setState({
+      selectedUserId: 0,
+      selectedUser: {},
```

Comment:
```suggestion
      selectedUser: null,
```
use null for reset value